pkgs/development/tools/qtcreator/default.nix

```diff
@@ -28,7 +28,7 @@ mkDerivation rec {
     sha256 = "07i045mzwbfhwj2jlijhz9xs6ay03qs5dgcw2kzlcr79a69i0h6j";
   };
 
-  buildInputs = [ qtbase qtscript qtquickcontrols qtdeclarative ] ++
+  buildInputs = [ qtbase qtscript qtquickcontrols qtdeclarative elfutils.dev ] ++
```

Comment:
```suggestion
  buildInputs = [ qtbase qtscript qtquickcontrols qtdeclarative elfutils ] ++
```

Reply:
Didn't work because the default has the binaries